03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,57 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <!-- You code start here -->
+    <main>
+       <header>
+        <h1>The Daily Planet</h1>
+
+
+    <nav>
+        <a href="/html/">DESIGN</a> |
+        <a href="/css/">TECH</a> |
+        <a href="/js/">COOL</a> |
+        <a href="/jquery/">FUN</a>
+    </nav>
+     </header>
+    <section>
+        <artical>
+            <header>
+                <h1>When Zika Got Political, American Cities Got Screwed</h1>
+                <p>AUTHOR: EMMA GREY ELLIS.</p>
+            </header>
+            <img src="https://www.wired.com/wp-content/uploads/2016/08/GettyImages-584696822-1024x683.jpg" alt="Miami-Dade Mosquito Control Works In Area Where Zika Has Been Found">
+            <br>
+            <br>
+            <p>THE ZIKA VIRUS has officially arrived in the United States, as experts always knew it would. Florida Governor Rick Scott has announced that 15 people have contracted Zika from local mosquitoes in Miami. In response, the CDC put out a travel
+                advisory warning pregnant women against traveling to the Zika-affected part of Miami—the first time such a warning has ever been applied to part of the continental US. And to combat the virus, Scott has requested a CDC Emergency Response
+                Team to support the state’s research and blood sample collection.
+                <br>
+                <br>
+                While that’s a good start—and Scott has been getting some criticism for not calling on the CDC sooner—it’s kind of like slapping a Band-Aid on a severed artery. An emergency
+                response team simply isn’t built to attack the threat of Zika, and it’s no replacement for the federal government’s long-promised emergency funding.</p>
+        </artical>
+    </section>
+    <aside>
+<a href="http://www.coca-cola.com/global/glp.html">
+      <img src="https://upload.wikimedia.org/wikipedia/commons/2/2c/Flasche_Coca-Cola_0,2_Liter.jpg">
+    </a>
+    </aside>
+    </main>
+    <footer>
+      <img src="http://www.alt-codes.net/images/copyright-symbol.png"/>
+      <img src="https://cdn0.iconfinder.com/data/icons/social-flat-rounded-rects/512/facebook-256.png"/>
+      <img src="http://icons.iconarchive.com/icons/designbolts/minimalist-social/256/Twitter-2-icon.png"/>
+
+    </footer>
+
+
+</body>
+
+</html>

```

Comment:
Thank you for your reminding!
I already modified them. Please let me if you have any questions.

Best,
Xun Meng


> On Aug 4, 2016, at 15:58, Ethan Stenis <notifications@github.com> wrote:
> 
> In 03LessonThree/in-class-project/daily-planet.html <https://github.com/AustinCodingAcademy/intro-workbook/pull/91#discussion_r73601573>:
> 
> > +<a href="http://www.coca-cola.com/global/glp.html">
> > +      <img src="https://upload.wikimedia.org/wikipedia/commons/2/2c/Flasche_Coca-Cola_0,2_Liter.jpg">
> > +    </a>
> > +    </aside>
> > +    </main>
> > +    <footer>
> > +      <img src="http://www.alt-codes.net/images/copyright-symbol.png"/>
> > +      <img src="https://cdn0.iconfinder.com/data/icons/social-flat-rounded-rects/512/facebook-256.png"/>
> > +      <img src="http://icons.iconarchive.com/icons/designbolts/minimalist-social/256/Twitter-2-icon.png"/>
> > +
> > +    </footer>
> > +
> > +
> > +</body>
> > +
> > +</html>
> Great job, Xun! Your eye for design and layout is PRO! Just one small project requirement you left out: "Add copyright and company name to footer."
> 
> If you can add this in and commit again, I can bump your grade up to 100.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/AustinCodingAcademy/intro-workbook/pull/91/files/b0e9973f854aed58fde809476d8a7dc3e031875a..b53a2364772c4b139435d4d7e57da015fdc0fbf0#r73601573>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AMUUk0WQM4jZhlbV0j-7sRxGAyWEfz-Pks5qclKOgaJpZM4JVvbK>.
>